Updates to the data manipulation slides.

diazrenata opened the pull request:
- Adds a section on "good code"
- Trims the part where we talk about data types
- Re-focuses homework to emphasize colloquium project.

diazrenata committed:
Updates to the data manipulation slides.

Aariq commented on lessons/6-data-manipulation/slides.qmd:
```diff
@@ -69,7 +96,7 @@ We might need to create a repo for R scripting for this workshop.
 -   ‼️ Extra installs over "base R"
 -   ‼️ Very different syntax than "base R"
--   ‼️ Updates rapidly
+-   ‼️ Updates rapidly (can be a reproducibility challenge)
```
You might note (on the slides or just verbally) that the tidyverse _is_ pretty exceptional when it comes to having helpful warning and error messages that help you update your code when you need to.  Actually _breaking_ changes are pretty rare, and when they do happen the error messages usually give you instructions on how to update your code.

Aariq commented on lessons/6-data-manipulation/slides.qmd:
```diff
@@ -25,32 +25,59 @@ Do you collaborate with anyone who could or currently uses GitHub?
+## GitHub practice
 
-## Warm-up
+Create a repo for your notes and code from the rest of the workshop.
```
I think they will already have one, as we discussed on Slack.

Aariq reviewed (approved):
Looks good to me!

diazrenata committed:
Correct incrementals and rerender using updated quarto.

diazrenata committed:
update from main

Merge branch 'main' of https://github.com/cct-datascience/repro-data-sci into data-manip

# Conflicts:
#	_freeze/site_libs/revealjs/dist/theme/quarto.css

diazrenata committed:
get quarto.css from main
